### DISCUSSION TITLE
Automatic adding?

### DISCUSSION THREAD

**Discussion by Josh5A** [apprenticeharper/DeDRM_tools#1646] [Q&A]
Recently lost my old system and didn't have a proper backup…

With my previous setup, Hazel would automatically copy downloaded Kindle books into a folder that Calibre monitored for automatic adding. DeDRM would do its magic, and Calibre would convert the Kindle books to epubs.

Just installed the latest Calibre and DeDRM and this is no longer working —*the Kindle books are not having the DRM removed. DeDRM works if I manually import them.

Is there any way to make this work?

**Comment by ElleKayEm** [comment]
Possibly if you go back to Kindle 1.17 where books are downloaded to one folder and no KFX format.
